src/libaktualizr/storage/sqlstorage.cc

```diff
@@ -352,22 +352,15 @@ void SQLStorage::storeMetadata(const Uptane::MetaPack& metadata) {
     return;
   }
 
-  if (db.exec("DELETE FROM meta;", nullptr, nullptr) != SQLITE_OK) {
+  if (db.exec("DELETE FROM " + tablename + ";", nullptr, nullptr) != SQLITE_OK) {
```

Comment:
The rule for APIs is that it should as hard as possible to abuse them. This should either use a tool to do SQL binding of params, or it should have a manual check that the table name matches `[a-zA-Z_][0-9a-zA-Z_]*` just to be super sure nothing bad happens.

Reply:
I could add the check, but why I thought it would be fine here is because storeMetadataCommon is a private function, not a part of interface. And `tablename` can only take one of two hardcoded strings. So the check would be like our code checking data from itself, which doesn't make much sense. We could, for instance, use some enum variable instead of `tablename` and then infer tablename in this function internally in a switch, but it will make the code somewhat more verbose.

Reply:
This could be done with an enum as an argument that maps it to the table. That would prevent it from passing a bad string on accident.